### DISCUSSION TITLE
Dark Energy: Could quantum gravity models arising from holography explain cosmological acceleration?

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#68] [Gravitation Models]
"*Could quantum gravity models arising from holography explain cosmological acceleration?*"
[https://phys.org/news/2023-06-quantum-gravity-holography-cosmological.html](https://phys.org/news/2023-06-quantum-gravity-holography-cosmological.html)

Assumptions that are added by this model: (***highlighted***)
1. "*the gravitational physics is encoded in a simpler, lower dimensional non-gravitational quantum system*" as a ***hologram***,
2. "*... explain cosmic acceleration ... via a dark energy whose* ***density changes with time***",
3. "*at some point the density of dark energy (DE) decreases below zero,* ***reaching a negative value***", i.e. a negative DE density is claimed to make sense.

Assumptions that are abandoned by this model:
[***none***]

This model can fit data better than our current model, but since it uses three additional assumptions, this is a moot point.  More epicycles always fit the data better.

This idea is dismissed.

**Comment by HanDeBruijn** [comment]
Sounds like: [Erik Verlinde Public Lecture: A New View on Gravity and the Dark Side of the Cosmos](https://www.youtube.com/watch?v=-OlEGq_g2FI) . More about Erik Verlinde's work is found in Wikipedia: [Entropic gravity](https://en.wikipedia.org/wiki/Entropic_gravity) . No coincidence perhaps that one of the researchers (Mark van Raamsdonk) who carried out the study has a Dutch name too.
